pilot/pkg/features/pilot.go

```diff
@@ -400,4 +400,8 @@ var (
 		15*time.Second,
 		"If set, the max amount of time to delay a push by. Depends on PILOT_ENABLE_FLOW_CONTROL.",
 	).Get()
+
+	PilotEnableLoopBlockers = env.RegisterBoolVar("PILOT_ENABLE_LOOP_BLOCKER", true,
+		"If enabled, Envoy will be configured to prevent traffic directly the the inbound/outbound "+
```

Comment:
"Envoy will be configured to prevent traffic directly" - This is confusing. Can we change to "Envoy will be configured to not send traffic directly to inbound/outbound ports" ?

Reply:
How about "PILOT_DENY_TRAFFIC_ON_LISTEN_PORTS" or "PILOT_DISABLE_TRAFFIC_ON_LISTEN_PORTS" ?

Reply:
Seems doesn't matter, it will be removed